src/name/verify.rs

```diff
@@ -31,6 +31,7 @@ pub fn verify_cert_dns_name(
     iterate_names(
         Some(cert.subject),
         cert.subject_alt_name,
+        false, // subject commonName never treated as dNSName
```

Comment:
Maybe name this with a `const` to better integrate the meaning?

Reply:
I think I will replace this bool with an enum.